packages/release-please/src/release-please.ts

```diff
@@ -520,6 +522,49 @@ const handler = (app: Probot) => {
         octokit,
         {logger, skipPullRequest: branchConfiguration.onDemand}
       );
+      if (branchConfiguration.tagPullRequestNumber) {
+        context.payload.commits.forEach(c => commits.add(c.id));
```

Comment:
If we don't actually need all of the commits, consider just grabbing a count (or boolean for > 1)